rdtools/plotting.py

```diff
@@ -2,9 +2,9 @@
 
 
 def degradation_summary_plots(yoy_rd, yoy_ci, yoy_info, normalized_yield,
-                              hist_xmin=None, hist_xmax=None, scatter_ymin=None,
-                              scatter_ymax=None, plot_color=None, summary_title=None,
-                              scatter_alpha=0.5):
+                              hist_xmin=None, hist_xmax=None,  bins=75, 
+                              scatter_ymin=None, scatter_ymax=None, plot_color=None, 
+                              summary_title=None, scatter_alpha=0.5):
```

Comment:
I'm generally in favor with adding this control of the bin count. This does also change the default behavior from `yoy_values//40` to 75. I want to make sure everyone agrees. @dirkjordan ? 

If not we can make the default `None` with a conditional that sets bins to `yoy_values//40`